frontend/src/pages/petlistings/PetDetailCreate.jsx

```diff
@@ -0,0 +1,365 @@
+
+                            <div className="pet-profile-details edit-mode">
+                                <div className="pet-profile-header edit-mode">
+                                    <input className="title" name="name" value={listing.name} onChange={handlePetChange} placeholder="pet name" required />
+                                    <input readOnly id="address" className="address" name="address" value={shelter.address} onChange={handleShelterChange} placeholder='shelter address' />
+                                </div>
+                                <div className="pet-profile-info">
+                                    <div className="row">
+                                        <label htmlFor="shelter" className="key">Shelter:</label>
+                                        <input readOnly name="email" className="value" value={shelter.email} onChange={handleShelterChange} placeholder='shelter email' />
+                                    </div>
+                                    <div className="row">
+                                        <label htmlFor="size" className="key">Size:</label>
+                                        <input className="value" name="size" value={listing.size} onChange={handlePetChange} />
+                                    </div>
+                                    <div className="row">
+                                        <label htmlFor="breed" className="key">Breed:</label>
+                                        <input className="value" name="breed" value={listing.breed} onChange={handlePetChange} />
+                                    </div>
+                                    <div className="row">
+                                        <label htmlFor="age" className="key">Age:</label>
+                                        <input type='number' className="value" name="age" value={listing.age} onChange={handlePetChange} />
+                                    </div>
+                                    <div className="row">
+                                        <label htmlFor="gender" className="key">Sex:</label>
+                                        <input name="gender" className="value" value={listing.gender} onChange={handlePetChange} />
+                                    </div>
+                                    <div className="row">
+                                        <label htmlFor="pet-status" className="key">Status:</label>
+                                        <input id="pet-status" className="value" name="status" value={listing.status} onChange={handlePetChange} />
+                                    </div>
+                                    <div className="row">
+                                        <label htmlFor="category" className="key">Category:</label>
+                                        <input id="category" className="value" name="category" value={listing.category} onChange={handlePetChange} />
```

Comment:
Can these fields (plus the size one above) be converted into drop downs instead or have the options given inside the placeholder ? It is slightly confusing to use if I don't know what's the fix set that I should be entering

Reply:
implemented as radio